hypervisor/arch/x86/reloc.c

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
This file works at boot time only, so move to boot directory.

Reply:
Sure, I'll move it to boot directory.